WPMedia/arc-themes-components
# THEMES-1904: Linting fixes for components

## Description

#### Jira Ticket: [THEMES-1904](https://arcpublishing.atlassian.net/browse/THEMES-1904)

Fixed the stylelint config for this repo, as well as linting errors preventing the status checks from passing. Some CSS rules were added to be ignored in the `.stylelintrc.json` since they were throwing errors on allowed Sass functions and controls.

## Test Steps

1. Checkout this branch - `git checkout THEMES-1904`
2. Update dependencies - `npm i`
3. Run the linting scripts `npm run lint && npm run lint:styles`
4. There should be no ESLint errors and nothing reported by Stylelint.


[THEMES-1904]: https://arcpublishing.atlassian.net/browse/THEMES-1904?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
